py-scripts/lf_webpage.py

```diff
@@ -116,10 +116,10 @@
 class HttpDownload(Realm):
```

Comment:
Same comment as previous PR. Was this tested with virtual clients? Can you please add CLI for virtual clients as well?